src/main/resources/templates/gcc/parent-info-review.html

```diff
@@ -1,24 +1,63 @@
 <!DOCTYPE html>
 <html th:lang="${#locale.language}" xmlns:th="http://www.thymeleaf.org">
-<head th:replace="~{fragments/head :: head(title='TEMPLATE')}"></head>
+<head th:replace="~{fragments/head :: head(title=#{parent-info-review.title})}"></head>
 <body>
 <div class="page-wrapper">
   <div th:replace="~{fragments/toolbar :: toolbar}"></div>
   <section class="slab">
     <div class="grid">
       <div th:replace="~{fragments/goBack :: goBackLink}"></div>
       <main id="content" role="main" class="form-card spacing-above-35">
-        <th:block th:replace="~{fragments/cardHeader :: cardHeader(header=#{placeholder}, subtext=#{placeholder})}"/>
-        <th:block th:replace="~{fragments/form :: form(action=${formAction}, content=~{::formContent})}">
-          <th:block th:ref="formContent">
-            <div class="form-card__content">
-                <!-- Put inputs here -->
+        <th:block th:replace="~{fragments/gcc-icons :: evaluatingInformation}"></th:block>
+        <th:block th:replace="~{fragments/cardHeader :: cardHeader(header=#{parent-info-review.header})}"/>
+          <div class="form-card__content">
+            <p class="text--left"><strong th:text="#{parent-info-review.personal.information}"></strong></p>
+            <ol class="list--bulleted">
+              <li th:text="${inputData.get('parentFirstName') + ' ' + inputData.get('parentLastName')}"></li>
+              <li th:text="${inputData.get('parentBirthMonth') + '/' + inputData.get('parentBirthDay') + '/' + inputData.get('parentBirthYear')}"></li>
+            </ol>
+            <a id="edit-parent-info-basic" th:href="'/flow/' + ${flow} + '/parent-info-basic-1'" th:text="#{parent-info-review.personal.information.link}"></a>
+            <hr class="spacing-below-15 spacing-above-15">
+
+            <p class="text--left"><strong th:text="#{parent-info-review.home.address}"></strong></p>
+            <div th:if="${inputData.containsKey('homeAddressLines') and !inputData.get('homeAddressLines').isEmpty()}">
```

Comment:
Let's check for checkbox input incase they go back in the flow
```suggestion
            <div th:unless="${T(org.ilgcc.app.utils.SubmissionUtilities).parentIsExperiencingHomelessness(fieldData)}">
```